config/v1/types_infrastructure.go

```diff
@@ -182,6 +182,25 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// provisioningInterface is the name of the network interface on a Baremetal server connected
+	// to the provisioning network.
+	ProvisioningInterface string `json:"provisioningInterface"`
+
+	// provisioningIp is the VIP to be used by the provisioning service. This must be on the provisioning
+	// subnet, and outside of the DHCP range.
```

Comment:
required or optional?

Reply:
This should say just "IP" not VIP, it's used to bring up a NIC for provisioning.

The provisioning interface is probably required in this API I think, as although there are defaults specified in the underlying containers, it's likely the MAO will require these values to set up the ConfigMap @sadasu ?

Reply:
OK, I thought this was the IP that floated around with the metal3 pod if it was rescheduled. Is that not the case?

Reply:
@deads2k All the config that is being added here are required for a Baremetal IPI deployment.

Reply:
@hardys ProvisioningInterface is a required parameters. We cannot makes any good assumptions about which interface on the physical server would be connected to the provisioning interface.